frontend/package.json

```diff
@@ -153,12 +153,13 @@
     "@patternfly/react-charts": "7.2.2",
     "@patternfly/react-component-groups": "^5.2.0",
     "@patternfly/react-console": "5.1.0",
-    "@patternfly/react-core": "^5.3.0",
-    "@patternfly/react-icons": "^5.3.0",
+    "@patternfly/react-core": "5.4.0",
```

Comment:
Removing the caret [has been problematic in the past](https://github.com/openshift/console/pull/13588#discussion_r1483491836), but does not appear to be here.  @vojtechszocs, WDYT?

Reply:
I'm OK with it as long as the Yarn lock file does not end up with multiple resolutions for these dependencies.

Reply:
Adding the caret in the past may have resolved the issue if a dependency needed a newer version of the pinned package (and the rest of the dependencies were also okay with that newer version, so it only pulled in the newer version).

I'm not sure if that is causing the test failure on CI though, having some trouble parsing what is going wrong there.

Reply:
> I'm not sure what is causing the test failure on CI though, having some trouble parsing what is going wrong there.

If you view [the build log](https://storage.googleapis.com/test-platform-results/pr-logs/pull/openshift_console/14428/pull-ci-openshift-console-master-frontend/1849126074366038016/build-log.txt), you'll see the following:

```
i18n files are not up to date. Run 'yarn i18n' then commit changes.
```

Your changes result in a diff to the i18n files, so they need to be regenerated and the diff committed.